src/modules/45-dashboards/strings/strings.en.yaml

```
@@ -43,6 +43,10 @@ editModal:
   editDashboard: 'Edit Dashboard'
   submitFail: 'Edit dashboard failed, Please try again.'
   success: 'Successfully updated the dashboard'
+cloneDashboardModal:
+  title: 'Clone Dashboard'
+  submitFail: 'Clone dashboard failed, Please try again.'
+  success: 'Successfully cloned the dashboard'
```

Not sure what your tech writer would say, but I guidance from UX is to capitalise all Harness key words
```suggestion
  submitFail: 'Clone Dashboard failed, Please try again.'
  success: 'Successfully cloned the Dashboard'
```